src/Camera.d.ts

```diff
@@ -4,6 +4,7 @@ import { Orientation } from './index';
 export type OnReadCodeData = {
   nativeEvent: {
     codeStringValue: string;
+    codeFormat: string;
```

Comment:
The expected strings are known, could you use the enum notation instead?

// types.ts
export type CodeFormat = 'code-128' | 'code-39' | ...;

// Camera.d.ts
codeFormat: CodeFormat;

Reply:
Added this on commit 4af9bac